ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -65,7 +68,10 @@ const PermitSimulation: React.FC = () => {
                 paddingInline={2}
                 textAlign={TextAlign.Center}
               >
-                {value}
+                {formatNumber(
+                  value / Math.pow(10, tokenDecimals),
```

Comment:
Rather than doing this math inline, is it worth including this within the `formatNumber` util or a new one?

We seem to also do this within `dataTree`.

Maybe a combined function such as `applyDecimals`?

Reply:
I think it will not be very useful to do division inside format number to get actual value of tokens. It is not a lot of calculation thus I I left it inline.

Reply:
It is indeed brief, but I'd argue it still limits the readability a little, plus introduces duplication.

It could be both the division and the `pow` call also.

Reply:
It looks bit like over-engineering to me to add a util method for this.